ocs_ci/ocs/resources/pod.py

```diff
@@ -2263,3 +2270,99 @@ def get_crashcollector_pods(
     namespace = namespace or config.ENV_DATA["cluster_namespace"]
     crashcollectors = get_pods_having_label(crashcollector_label, namespace)
     return [Pod(**crashcollector) for crashcollector in crashcollectors]
+
+
+def check_pods_in_statuses(
+    expected_statuses,
+    pod_names=None,
+    namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+    raise_pod_not_found_error=False,
+):
+    """
+    checks whether the pods in a given namespace are in the expected statuses or not
+
+    Args:
+        expected_statuses (list): The expected statuses of the pods
+        pod_names (list): List of the pod names to check.
+            If not provided, it will check all the pods in the given namespace
+        namespace (str): Name of cluster namespace(default: defaults.ROOK_CLUSTER_NAMESPACE)
+        raise_pod_not_found_error (bool): If True, it raises an exception, if one of the pods
+            in the pod names are not found. If False, it ignores the case of pod not found and
+            check the pod objects of the rest of the pod names. The default value is False
+
+    Returns:
+        Boolean: True, if the pods are in the expected statuses. False, otherwise
+
+    """
+    if pod_names:
+        list_of_pods = get_pod_objs(pod_names, raise_pod_not_found_error)
```

Comment:
I would pass the `namespace` to the `get_pod_objs(...)` here too.

Reply:
Yes, np.